api.php

```diff
@@ -86,8 +87,44 @@
 		->withStatus(200);
 });
 
+//finds User by Email TODO modify route
+$app->get('/user/{email}', function (Request $request, Response $response, array $args) {
+	$email = $args['email'];
+	if (!$email) {
```

Comment:
What is this check doing? No comment to explain.

Reply:
commit #1ebe297